gensn/distributions.py

```diff
@@ -293,3 +293,39 @@ def __init__(self, rate=None, _parameters=None, event_dims=1):
             **kwargs,
             _parameters=_parameters,
         )
+
+
+class IndependentHalfNormal(WrappedTrainableDistribution):
```

Comment:
Could you put a description about the new distributions?

Reply:
Done. Could you please check below?